converters/fromZigbee.js

```diff
@@ -3480,14 +3484,84 @@ const converters = {
         cluster: 'hvacThermostat',
         type: ['readResponse'],
         convert: (model, msg, publish, options, meta) => {
-            const lookup = {0: 'unoccupied', 1: 'occupied'};
-            const lookup1 = {0: 'on_demand', 1: 'sensing'};
             const result = {};
+            const cycleOutputLookup = {15: '15 sec', 300: '5 min', 600: '10 min',
+                900: '15 min', 1200: '20 min', 1800: '30 min', 65535: 'off'};
             if (msg.data.hasOwnProperty('1024')) {
+                const lookup = {0: 'unoccupied', 1: 'occupied'};
                 result.thermostat_occupancy = lookup[msg.data['1024']];
             }
+            if (msg.data.hasOwnProperty('1025')) {
+                result.main_cycle_output = cycleOutputLookup[msg.data['1025']];
+            }
             if (msg.data.hasOwnProperty('1026')) {
-                result.backlight_auto_dim = lookup1[msg.data['1026']];
+                const lookup = {0: 'on_demand', 1: 'sensing'};
+                result.backlight_auto_dim = lookup[msg.data['1026']];
+            }
+            if (msg.data.hasOwnProperty('1028')) {
+                result.aux_cycle_output = cycleOutputLookup[msg.data['1028']];
+            }
+            return result;
+        },
+    },
+    sinope_TH1400ZB_specific: {
+        cluster: 'manuSpecificSinope',
+        type: ['readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('outdoorTempToDisplay')) {
+                result.outdoor_temp_to_display = msg.data['outdoorTempToDisplay'];
```

Comment:
`outdoor_temp_to_display` -> `outdoor_temperature_to_display`

Reply:
Attribute removed. I forgot to remove it.